secretsanta.txt

```diff
@@ -0,0 +1,9 @@
+<p>You have been randomly selected to be the secret santa for {}.</p>
```

Comment:
Just passing through. 🙂 This looks great @Gathondu ! 👍🏾 🎉  Might we check with PnC to confirm the message structure? cc @austinroy

Reply:
@akabiru Thanks. That message was from PnC. We had a different message structure but PnC advised on this format.